### DISCUSSION TITLE
re: Joining forces with NuShell

### DISCUSSION THREAD

**Discussion by good-unit** [johnkerl/miller#1237] [Ideas]
Have you considered joining the NuShell (https://www.nushell.sh/) project?
NuShell provides all your projects' functionality and more -- and is quickly growing day by day.

Joining forces would get things done quicker, centralize energy, stop fragmentation in this space.

Thanks.

**Comment by trantor** [comment]
mmm it's actually grown quite a lot since I've looked into it years ago...

**Reply by johnkerl** [reply]
Indeed @trantor it's amazing!!
